lbz/handler_base.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from typing import Generic, TypeVar
+
+from lbz.misc import deprecated
+
+T = TypeVar("T")
+
+
+class HandlerBase(Generic[T]):
```

Comment:
As it is intended to not use it directly, you should mark it as an abstract base class:
```suggestion
class HandlerBase(Generic[T], metaclass=ABCMeta):
```